Create harbour-ieligmmhg-hu.ts

1Zgp opened the pull request:
Ungarische Übersetzung für deine App.

Utkiek commented:
Hi,

thanks for your translations. Sorry for little late but now I have updated the repository on github and merged your translation. Unfortunaly some translations are missing in the today version. You are invited to update the translation.

For a first impression how your old translation works you can install a test version (with hungary translation):
[https://oc.plutex.de/s/s8Bdb3KAMNCtmWs](https://oc.plutex.de/s/s8Bdb3KAMNCtmWs)
Password: CcyjWpzwHL

The github repositories for my other 2 apps are not upto date. I will do this in the next days..

Thanks and greetings from Bremen
Hans

1Zgp commented:
Hi Hans,
I updated the Hungarian translation file. 
About testing: as currently I am using an iPhone I can’t test the app (I had an Xperia x with Sailfish on it, now I am waiting for the OS to be released for Xperia 10 mark 3. Till then I cannot use Sailfish apps). I make translations for apps I think are useful for the community and for me as well. 
Thank you for your work!

> 2022. jan. 25. dátummal, 19:29 időpontban Utkiek ***@***.***> írta:
> 
> ﻿
> Hi,
> 
>  thanks for your translations. Sorry for little late but now I have updated the repository on github and merged your translation. Unfortunaly some translations are missing in the today version. You are invited to update the translation.
> 
> For a first impression how your old translation works you can install a test version (with hungary translation):
> https://oc.plutex.de/s/s8Bdb3KAMNCtmWs
> Password: CcyjWpzwHL
> 
> The github repositories for my other 2 apps are not upto date. I will do this in the next days..
> 
> Thanks and greetings from Bremen
> Hans
> 
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> Triage notifications on the go with GitHub Mobile for iOS or Android. 
> You are receiving this because you authored the thread.
